evap/rewards/tools.py

```diff
@@ -91,8 +97,14 @@ def grant_reward_points_after_evaluate(sender, **kwargs):
     request = kwargs['request']
     semester = kwargs['semester']
 
-    if grant_reward_points(request.user, semester):
-        messages.success(request, _("You just have earned reward points for this semester because you evaluated all your courses. Thank you very much!"))
+    result = grant_reward_points(request.user, semester)
+    if result:
+        granted, remaining = result
+        reward = ngettext("{count} reward point", "{count} reward points", granted).format(count=granted)
+        if remaining:
+            messages.success(request, _("You just earned {reward} for this semester. More points can be earned by evaluating more courses.").format(reward=reward))
```

Comment:
can you please also take into account whether the user can actually evaluate more courses? i.e. there are courses the user didn't evaluate and which have an evaluation period that ends in the future.
because the user might just have not participated in some already finished evaluations and would then have no chance to get more points.

Reply:
Well right, that actually makes it much more difficult...